one/src/daemon.rs

```diff
@@ -168,6 +168,14 @@ pub struct DaemonOpts {
         requires = "experimental_features"
     )]
     event_validation: bool,
+
+    /// Flight SQL bind address; Requires using the experimental-features flag
```

Comment:
do we want a separate flag for setting the port number?

Reply:
I prefer fewer flags, this handle both host and port in a single flag.